<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiffieHellman</class>
 <widget class="QWidget" name="DiffieHellman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel{
font-family:&quot;Microsoft YaHei&quot;;
font-size:30px;
font-weight:bold;
color:white;
border:none;
font-style:italic;
}
</string>
     </property>
     <property name="text">
      <string>Diffie-Hellman密钥交换 </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">QLabel{
font-family:&quot;Microsoft YaHei&quot;;
font-size:20px;
font-weight:bold;
color:white;
border:none;
}
</string>
        </property>
        <property name="text">
         <string>请选择一个素数：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="sushuEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_4">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color:#473C8B;
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:#8B7355;
font:bold 14px;
min-width:2em;
color:white;
font-family:华文新魏;
padding:5px;
font-size:20px
}
QPushButton:Pressed
{
background-color:rgba(100,255,100,200);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}
QPushButton:hover
{
background-color:rgba(100,255,100,100);
border-color:rgba(255,255,255,200);
color:rgba(0,0,0,200);
}</string>
        </property>
        <property name="text">
         <string>Eve干扰</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color:#473C8B;
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:#8B7355;
font:bold 14px;
min-width:2em;
color:white;
font-family:华文新魏;
padding:5px;
font-size:20px
}
QPushButton:Pressed
{
background-color:rgba(100,255,100,200);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}
QPushButton:hover
{
background-color:rgba(100,255,100,100);
border-color:rgba(255,255,255,200);
color:rgba(0,0,0,200);
}</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="yuangenEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">QLabel{
font-family:&quot;Microsoft YaHei&quot;;
font-size:20px;
font-weight:bold;
color:white;
border:none;
}
</string>
        </property>
        <property name="text">
         <string>原根：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">QLabel{
font-family:&quot;Microsoft YaHei&quot;;
font-size:20px;
font-weight:bold;
color:white;
border:none;
}
</string>
        </property>
        <property name="text">
         <string>Alice：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color:#473C8B;
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:#8B7355;
font:bold 14px;
min-width:2em;
color:white;
font-family:华文新魏;
padding:5px;
font-size:20px
}
QPushButton:Pressed
{
background-color:rgba(100,255,100,200);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}
QPushButton:hover
{
background-color:rgba(100,255,100,100);
border-color:rgba(255,255,255,200);
color:rgba(0,0,0,200);
}</string>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">QLabel{
font-family:&quot;Microsoft YaHei&quot;;
font-size:20px;
font-weight:bold;
color:white;
border:none;
}
</string>
        </property>
        <property name="text">
         <string>Bob：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color:#473C8B;
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:#8B7355;
font:bold 14px;
min-width:2em;
color:white;
font-family:华文新魏;
padding:5px;
font-size:20px
}
QPushButton:Pressed
{
background-color:rgba(100,255,100,200);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}
QPushButton:hover
{
background-color:rgba(100,255,100,100);
border-color:rgba(255,255,255,200);
color:rgba(0,0,0,200);
}</string>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_2"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
